Octokit/Models/Request/PullRequestRequest.cs

```diff
@@ -13,15 +13,15 @@ public class PullRequestRequest : RequestParameters
     {
         public PullRequestRequest()
         {
-            State = ItemState.Open;
+            State = ItemStateFilter.Open;
             SortProperty = PullRequestSort.Created;
             SortDirection = SortDirection.Descending;
         }
 
         /// <summary>
-        /// "open" or "closed" to filter by state. Default is "open".
+        /// "open" , "closed" or "All" to filter by state. Default is "open".
```

Comment:
use lowercase for consistency, also there is an extraneious space space after "open"

eg:
> /// "open", "closed" or "all" to filter by state.  Default is "open".